owners.json

```diff
@@ -1,4 +1,5 @@
 {
+  "bbannier": {"slack": "benjamin", "jira": "benjamin.bannier"},
```

Comment:
I am not sure why we use handles for the Mesosphere-internal slack here, but this seems consistent with the rest of the file.

For posterity, my DC/OS slack handle would be `bbannier` and Till's is `till.mesosphere`.